content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -0,0 +1,96 @@
+
+Note that this information is documented during and in context of an encounter, may not be actively managed, and may not be consolidated across encounters as a "patient-level" plan.  Therefore, this data should be interpreted as a snapshot in time of a point in time.  For example, if a patient has an encounter related to injury due to a car accident, there will be a snapshot of this information in context of those injuries.  If there is a subsequent encounter related to an illness, there may be a snapshot of this data set related to that illness.  If a patient queries for the care plan resource a week after the last encounter, the response will be the latest instance of the last set of this data documented; i.e., the most recent charted version of the illness related data.  If the query includes a date range inclusive of both encounters, the response will include the most recent charted version of both the injury and illness data sets.  Both data sets will be the most recent data documented for the encounter, not of a specific dated instance, and, since in practice, they may not be actively managed, they may not reflect the current state of goals, concerns and plans; they may represent the initially documented state. For context, the narrative response will provide a charted date to represent to what date context this information is relevant.
+
+
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+
+## Search
+	
+	GET /CarePlan?:parameters
+	
+_Implementation Notes_
+
+* The API returns the most current careplan list.
+
+* The Narrative includes additional information pertaining to the careplan, and should be shown to the user to ensure completeness of clinical content and context.
+
+### Parameters
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patientId | Y | [reference](http://hl7.org/fhir/DSTU2/search.html#reference) | The patient identifier provided in a pre-requisite authorization step. |
+| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
+
+Notes:
+
+* See details regarding special [date](../../#special-information-regarding-date-parameters) requirements.
+* If the query date parameter is null in the query, the API will return qualifying records with dates less than or equal to the date of query as limited by the lesser of 1000 records or 30 days inclusive of the query date.  
+* If a single date parameter is included, the response will include qualifying records dated that day as limited by 1000 records.  
+* If a valid date range is used in the query, the API will return qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit date specified.
```

Comment:
Change to
If a valid date range is used in the query, the API will return qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit of the range.